common/evaluators/reuters_evaluator.py

```diff
@@ -0,0 +1,28 @@
+import torch
+import torch.nn.functional as F
+import numpy as np
+
+from .evaluator import Evaluator
+
+
+class ReutersEvaluator(Evaluator):
+
+    def get_scores(self):
+        self.model.eval()
+        self.data_loader.init_epoch()
+        n_dev_correct = 0
+        total_loss = 0
+
+        for batch_idx, batch in enumerate(self.data_loader):
+            scores = self.model(batch.text)
+            # Using binary accuracy
+            for tensor1, tensor2 in zip(scores.round().long().data, batch.label.data):
+                print(tensor1, tensor2)
+                if np.array_equal(tensor1, tensor2):
```

Comment:
So your evaluation criteria is that: give full credit when it is exact match otherwise, it get zero ?

Reply:
Yup. Since only one or two in 90 classes would be the positive label, I'm not sure if something like hamming score would work. The fraction of labels that are incorrectly predicted would still be too small in that case,